### DISCUSSION TITLE
terraform resource related to global accounts

### DISCUSSION THREAD

**Discussion by bilel-mrad** [wallix/terraform-provider-wallix-bastion#50] [Q&A] [answered]
Hey All,

I'm new to wallix and want to automate stuff related to wallix , I've found the wallix-bastion provider but struggling with resources .

What are the resources related to global accounts under targets/accounts ?

**Comment by bsimonWallix** [comment, marked as answer]
Please see the answer on an other thread: https://github.com/wallix/terraform-provider-wallix-bastion/issues/43

If you are new to WALLIX, please use the official documentation to help you out on the basic notion.
